File: C/Odev3-PDP/src/RastgeleKisi.c
/** 
 * 
 * @author Burak Yıldırım
 *         M.İbrahim Turan
 * @since 22.04.2018
 * <p>
 * TC kimlik ve IMEI Kontrolü yapan program simulasyonu
 * </p>
 */


#include "RastgeleKisi.h"
#include "stdio.h"
#include "stdlib.h"

RastgeleKisi kisiUret()
{
	RastgeleKisi r ;
	
	r =(RastgeleKisi)malloc(sizeof(struct RASTGELEKISI));
	r->kisiler = kisiOlustur();
	
	FILE* dosya ;
	dosya = fopen("Kisiler.txt" , "a");
	
	for(int i = 0 ; i < 11 ; i++)
	{
		fprintf(dosya,"%d",r->kisiler->kimlik->tcNO[i]);
	}
	fprintf(dosya," ");
	fprintf(dosya,r->kisiler->isimSoyisim);
	fprintf(dosya," ");
	fprintf(dosya,"%d",r->kisiler->yas);
	fprintf(dosya," ");
	
	for(int i = 0 ; i < 11 ; i++)
	{
		fprintf(dosya,"%d",r->kisiler->telefon->telNO[i]);
	}
	fprintf(dosya," ");
	fprintf(dosya,"(");
	for(int i = 0 ; i < 15 ; i++)
	{
		fprintf(dosya,"%d",r->kisiler->telefon->i->imeiNO[i]);
	}
	fprintf(dosya,")");
	fprintf(dosya,"\n");
	
	
	fclose(dosya);
	
}

File: JAVA/JavaOdev2/RASTGELEKISIURET/src/Kisiler/Rastgelekisi.java
/** 
 * 
 * @author Burak Yıldırım
 *         M.İbrahim Turan
 * @since 01.04.2018
 * <p>
 * TC kimlik ve IMEI Kontrolü yapan program simulasyonu
 * </p>
 */
package Kisiler;



import java.io.*;


public class Rastgelekisi 
{
    
   
      
   public String RastgeleKisiUret() throws FileNotFoundException,IOException
   {
       Kisi kisiler = new Kisi();
       kisiler.kimlik.tcNoUret();
       String tcNO = "";
       for(int i =0 ; i< kisiler.kimlik.tcNoGetir().length ; i++)
       {
           tcNO = tcNO + Integer.toString(kisiler.kimlik.tcNoGetir()[i]) ;
       }
       kisiler.telefon.imei.imeiUret();
       String imei = "" ;
       for(int i =0 ; i< kisiler.telefon.imei.imeiNOGetir().length ; i++)
       {
           imei = imei + Integer.toString(kisiler.telefon.imei.imeiNOGetir()[i]) ;
       }
       kisiler.telefon.telNoUret();
       String telNO = "";
       for(int i =0 ; i< kisiler.telefon.telNoGetir().length ; i++)
       {
           telNO = telNO + Integer.toString(kisiler.telefon.telNoGetir()[i]) ;
       }
       
       kisiler.yasAta();
       String yas = Integer.toString(kisiler.yasGetir());
       kisiler.isimAta();
       return tcNO + " " + kisiler.isimGetir() + " " + yas + " " + telNO + " " + "(" + imei + ")" ;
   } 
  
    
   
    
   
    
    
   
       
    
    
    
}
